test/JDBC/expected/non_default_server_collation/chinese_prc_ci_as/test_search_path.out

```diff
@@ -0,0 +1,188 @@
+-- psql
+ALTER SYSTEM SET babelfishpg_tsql.migration_mode = 'multi-db';
+SELECT pg_reload_conf();
```

Comment:
What's the difference between chinese_prc_ci_as output and japanese_ci_as output ? Sorry I can't review it line by line.

Reply:
The expected file for chinese_prc_ci_as and japanese_ci_as are same.